style.css

```diff
@@ -97,4 +99,210 @@ ul {
     max-width: 100%;
     height: auto;
   }
+
+  /* main section */
+
+  .work_section {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    justify-items: center;
+    background-color: white;
+    height: 100%;
+    row-gap: 1.625rem;
+    margin-bottom: 3%;
+  }
+
+  .work_section_title {
+    margin: 24% 0 10% 0;
+  }
+
+  /* styled the first box diffrently cause it did not had a bg img */
+  .first_box {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    word-wrap: break-word;
+    width: 375px;
+    height: 590px;
+    margin: 0 0 0 0;
+  }
+
+  .first_box h2 {
+    font-size: 1.5rem;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 100;
+    margin: 4% auto 0 7%;
+  }
+
+  .first_box p {
+    width: 88%;
+    padding-left: 1%;
+  }
+
+  .first_box_lang {
+    display: flex;
+    flex-flow: row;
+    justify-content: space-around;
+    margin-top: 1%;
+    margin-bottom: 4%;
+    margin-left: -14%;
+  }
```

Comment:
- [ ] Please 🙏🏽 add the correct styles, margin, and padding to be similar to the Figma design 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/323d3f50-2b4d-489a-b868-29d0f5d3fdd1) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/de3a9d8f-44f8-4a90-8107-3c2d6a427daa) |

Reply:
Done!